fetch.bs

```diff
@@ -6714,6 +6753,435 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+    <a for="fetch group">deferred fetch records</a>:</p>
+
+    <ol>
+     <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+     <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+     <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+     <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+     <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+     then decrement <var>quotaForRequestOrigin</var> by <var>requestLength</var>.
+    </ol>
+  </ol>
+
+ <li><p>If <var>quota</var> is equal or less than 0, then return 0.
+
+ <li><p>If <var>quota</var> is less than <var>quotaForRequestOrigin</var>, then return
+ <var>quota</var>.
+
+ <li><p>Return <var>quotaForRequestOrigin</var>.
+</ol>
+</div>
+
+<div algorithm>
+<p>To <dfn export>reserve deferred-fetch quota</dfn> for a <a>navigable container</a>
+<var>container</var> given an <a for=/>origin</a> <var>originToNavigateTo</var>:
+
+<p class=note>This is called on navigation, when the source document of the navigation is the
+<a for=/>navigable</a>'s parent document. It potentially reserves either 64kb or 8kb of quota for
+the container and its navigable, if allowed by permissions policy. It is not observable to the
+cotnainer document whether the reserved quota was used in practice. This algorithm assumes that the
+container's document might delegate quota to the navigated container, and the reserved quota would
+only apply in that case, and would be ignored if it ends up being shared. If quota was reserved and
+the document ends up being <a>same origin</a> with its parent, the quota would be
+<a data-lt="potentially free deferred-fetch quota">freed</a>.
+
+<ol>
+ <li><p>Set <var>container</var>'s <a>reserved deferred-fetch quota</a> to 0.
+
+ <li><p>Let <var>controlDocument</var> be <var>container</var>'s <a>node document</a>'s
+ <a>deferred-fetch control document</a>.
+
+ <li><p>If the <a data-lt="define an inherited policy for feature in container">inherited policy</a>
+ for "{{PermissionsPolicy/deferred-fetch}}", <var>container</var> and <var>originToNavigateTo</var>
+ is <code>Enabled</code>, and the <a>available deferred-fetch quota</a> for
+ <var>controlDocument</var> is equal or greater than
+ <a for="reserved deferred-fetch quota">normal quota</a>, then set <var>container</var>'s
+ <a>reserved deferred-fetch quota</a> to <a for="reserved deferred-fetch quota">normal quota</a> and
+ return.
+
+ <li>
+  <p>If all of the following conditions are true:
+
+  <ul class=brief>
+   <li><p><var>controlDocument</var>'s <a>node navigable</a> is a <a for=/>top-level traversable</a>
+
+   <li><p>The <a data-lt="define an inherited policy for feature in container">inherited policy</a>
+   for "{{PermissionsPolicy/deferred-fetch-minimal}}", <var>container</var> and
+   <var>originToNavigateTo</var> is <code>Enabled</code>
```

Comment:
```suggestion
   <li><p>the <a data-lt="define an inherited policy for feature in container">inherited policy</a>
   for "{{PermissionsPolicy/deferred-fetch-minimal}}", <var>container</var> and
   <var>originToNavigateTo</var> is <code>Enabled</code>; and
```
I don't like `Enabled` as an output. That's not Infra. Can you file an upstream issue?

Reply:
It should be strings actually, I'll amend.